CONTRIBUTING.md

```diff
@@ -70,6 +70,25 @@ contents of the `web-design-standards` project under `node_modules/uswds` within
 the `project-using-npm`. For more information on `npm-link`, you can read about
 it in the [`npm-link` documentation](https://docs.npmjs.com/cli/link).
 
+## Local Gem Development for the Jekyll Documentation Website
```

Comment:
Agree, "Jekyll Documentation Website" is pretty wordy. How about "styleguide website"? Also, agree that we maybe don't need Gem in the title. Also we use sentence case on titles, so only the first word should be capitalized.

Looping in our content extraordinaire, @kategarklavs.